__tests__/app.test.js

```diff
@@ -34,7 +34,59 @@ describe('GET /api/topics', () => {
       .get('/api/topics')
       .expect(200)
       .then(({ body }) => {
+        //Check that body contains a 'topics' key
         expect(body).toHaveProperty('topics')
+        expect(Array.isArray(body.topics)).toBe(true)
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: